src/processes/helper/sendTopicAutoMessage.js

```diff
@@ -33,12 +33,13 @@ const followTopicServiceQueue = async (userId, topicId, communityMessageFormatId
   }
 };
 const sendTopicAutoMessage = async (userId, topicIds) => {
+  console.log('::: sendTopicAutoMessage when register :::', userId);
   try {
     topicIds.forEach(async (topicId) => {
       const communityMessageFormats = await CommunityMessageFormat.findAll({
         where: {topic_id: topicId, status: '1'}
       });
-
+      console.log('::: Total communityMessageFormats :::', communityMessageFormats.length);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, consider adding a condition to only log the length of `communityMessageFormats` in a development environment.

```diff
- console.log('::: Total communityMessageFormats :::', communityMessageFormats.length);
+ if (process.env.NODE_ENV === 'development') {
+   console.log('::: Total communityMessageFormats :::', communityMessageFormats.length);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->